CATALOG-REFERENCE.md

```
@@ -217,11 +220,10 @@ see above). `Reads from` = the main catalog tables the view's SQL joins.
 > **materialized** `pg_stat[io]_all_*` snapshot tables; the `*_all_*` parents'
 > *live* SQL is what's broken (needs runtime stat functions - below).
 
-### Partial - executes but a column diverges (3)
+### Partial - executes but a column diverges (2)
 | View | Purpose | Reads from | Reflects | Diverging | Reason |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)